tools/Customization/DevHome.Customization/Views/FileExplorerPage.xaml.cs

```diff
@@ -82,4 +79,33 @@ public bool ValidateRepositoryPath(string rootPath)
 
         return true;
     }
+
+    private void AddRepository_Click(object sender, RoutedEventArgs e)
+    {
+        if (sender as Button is Button registerRepoButton)
```

Comment:
`sender is Button registerRepoButton` will work ya?  Why the `as`?

Reply:
Resolved in latest. Thanks!